spec/features/idv/steps/gpo_step_spec.rb

```diff
@@ -17,15 +17,15 @@
       and_return(days_passed + 1)
   end
 
-  it 'redirects to and completes the review step when the user chooses to verify by letter', :js do
+  it 'visit and complete the enter password step when the user chooses to verify by letter', :js do
```

Comment:
Rename gpo_step_spec.rb to request_letter_spec.rb while you're here?

Also, possible spec title edit here

```suggestion
  it 'visits and completes the enter password step when the user chooses to verify by letter', :js do
```